README.md

```diff
@@ -81,6 +81,19 @@ If you receive a "No machine capacity available" error message when submitting a
 
 FGMachine automatically reloads the `projects.json` file when it is changed.
 
+#### GPU capacity support
+
+In order to handle projects, which require GPUs to perform a task, you need to add two parameters for each project in `projects.json` file:
+
+```json
+{
+  "gpu_capacity": "<gpu capacity needed (as a fraction of one GPU capacity (e.g. 0.5)>",
+  "gpu_command": "<option to pass to script to identify card number (e.g. -gpu)>",
```

Comment:
Just to clarify, "option to pass to script to identify card number, including command line option style".